src/math/Kconfig

```diff
@@ -38,6 +38,15 @@ config SQRT_FIXED
 	  to  calculate square root.square function having positive number
 	  y as input and return the positive number x multiplied by itself (squared)
 
+config EXP_FIXED
+	bool "Exponential  functions"
+	default n
+	help
+	  By selecting this, the 32-bit exp_int32() function can be used to calculate
+	  exponential values. With a maximum ulp of 5, an exponential function with
+	  an input range of -5 to +5 y gives positive numbers between 0.0067379 and
+	  148.4131. The precision of this function is 1e-4.
```

Comment:
the number provided for range have a higher precision than the precision provided, is this intended?